src/main.rs

```diff
@@ -244,22 +239,17 @@ fn command_line() -> Commands {
                 }
 
                 #[cfg(debug_assertions)]
-                let allow_incompatible = std::env::var("SONARTEST_ROLLUP").is_ok();
+                let force_slurm = std::env::var("SONARTEST_ROLLUP").is_ok();
```

Comment:
These don't belong here, but before invoking ps, since they are no longer relevant to argument parsing.